service/matching/fx.go

```diff
@@ -55,6 +55,7 @@ var Module = fx.Options(
 	fx.Provide(RateLimitInterceptorProvider),
 	fx.Provide(HandlerProvider),
 	fx.Provide(service.GrpcServerOptionsProvider),
+	fx.Provide(TaskQueueReplicatorNamespaceReplicationQueueProvider),
```

Comment:
that's a mouthful :laughing:

Reply:
True, I'll address in #4259

Reply:
I wrote on the other one, but I just thought it was funny, didn't mean to suggest changing